src/quo2/components/keycard/view.cljs

```diff
@@ -0,0 +1,38 @@
+(ns quo2.components.keycard.view
+  (:require [react-native.core :as rn]
+            [quo2.components.keycard.style :as style]
+            [quo2.components.tags.tag :as tag]
+            [quo2.foundations.colors :as colors]
+            [status-im2.common.resources :as resources]
+            [utils.i18n :as i18n]))
+
+(defn keycard
+  "This component based on the following properties:
+  - :holder-name - Can be owner's name. Default is Empty
+  - :locked? - Boolean to specify whether the keycard is locked or not
+  "
+  [{:keys [holder-name? locked?]}]
```

Comment:
seems like `holder-name` should not have a question mark at the end of it. That is generally used in Clojurescript if the variable is a boolean but it seems that it is a string.